go/vt/mysqlctl/backup.go

```diff
@@ -344,21 +342,6 @@ func Restore(ctx context.Context, params RestoreParams) (*BackupManifest, error)
 		return nil, err
 	}
 
```

Comment:
given we are starting mysql above. It will always start in replica mode (super_read_only to true). I don't think this code block in any case will run for primary ...

Reply:
Since we no longer populate those metadata tables, it is safe to remove this.